solana-geyser-plugin-scaffold/src/fb_serializers/extractors.rs

```diff
@@ -1271,5 +1271,15 @@ fn extract_tx_status<'bldr: 'args, 'args: 'mut_bldr, 'mut_bldr>(
                 },
             ))
         }
+        solana_sdk::transaction::TransactionError::ProgramCacheHitMaxLimit => {
+            Some(TransactionError::create(
+                builder,
+                &TransactionErrorArgs {
+                    err_type: TransactionErrorType::UnbalancedTransaction,
```

Comment:
We need to add ProgramCacheHitMaxLimit variant into flatbuffer

Reply:
Should be done